docs/sources/project/make-a-contribution.md

```diff
@@ -78,315 +124,337 @@ To sync your repository:
-		   
-	This command says get all the changes from the `master` branch belonging to
-	the `upstream` remote.
-		   
+        $ git fetch upstream/master
+        remote: Counting objects: 141, done.
+        remote: Compressing objects: 100% (29/29), done.
+        remote: Total 141 (delta 52), reused 46 (delta 46), pack-reused 66
+        Receiving objects: 100% (141/141), 112.43 KiB | 0 bytes/s, done.
+        Resolving deltas: 100% (79/79), done.
+        From github.com:docker/docker
+           9ffdf1e..01d09e4  docs       -> upstream/docs
+           05ba127..ac2521b  master     -> upstream/master
+
+    This command says get all the changes from the `master` branch belonging to
+    the `upstream` remote.
+
 7. Rebase your local master with the `upstream/master`.
 
-		$ git rebase upstream/master
-		First, rewinding head to replay your work on top of it...
-		Fast-forwarded master to upstream/master.
-		
-	This command writes all the commits from the upstream branch into your local
-	branch.
-		
+        $ git rebase upstream/master
+        First, rewinding head to replay your work on top of it...
+        Fast-forwarded master to upstream/master.
+
+    This command writes all the commits from the upstream branch into your local
+    branch.
+
 8.  Check the status of your local branch.
 
-		$ git status
-		On branch master
-		Your branch is ahead of 'origin/master' by 38 commits.
-		  (use "git push" to publish your local commits)
-		nothing to commit, working directory clean
+        $ git status
+        On branch master
+        Your branch is ahead of 'origin/master' by 38 commits.
+          (use "git push" to publish your local commits)
+        nothing to commit, working directory clean
 
-	Your local repository now has any changes from the `upstream` remote.  You
-	need to push the changes to your own remote fork which is `origin/master`.
+    Your local repository now has any changes from the `upstream` remote.  You
+    need to push the changes to your own remote fork which is `origin/master`.
 
 9. Push the rebased master to `origin/master`.
 
-		$ git push
-		Username for 'https://github.com': moxiegirl
-		Password for 'https://moxiegirl@github.com': 
-		Counting objects: 223, done.
-		Compressing objects: 100% (38/38), done.
-		Writing objects: 100% (69/69), 8.76 KiB | 0 bytes/s, done.
-		Total 69 (delta 53), reused 47 (delta 31)
-		To https://github.com/moxiegirl/docker.git
-		   8e107a9..5035fa1  master -> master
+        $ git push
+        Username for 'https://github.com': moxiegirl
+        Password for 'https://moxiegirl@github.com': 
+        Counting objects: 223, done.
+        Compressing objects: 100% (38/38), done.
+        Writing objects: 100% (69/69), 8.76 KiB | 0 bytes/s, done.
+        Total 69 (delta 53), reused 47 (delta 31)
+        To https://github.com/moxiegirl/docker.git
+           8e107a9..5035fa1  master -> master
 
-	If you check 
+    If you check 
```

Comment:
Ok, this is a problem in my copy. Can you pull this sentence fragment while you are in there?